src/VisualStudio/Core/Def/Implementation/ProjectSystem/CPS/IWorkspaceProjectContext.cs

```diff
@@ -39,7 +37,7 @@ internal interface IWorkspaceProjectContext : IDisposable
         void AddAdditionalFile(string filePath, bool isInCurrentContext = true);
         void RemoveAdditionalFile(string filePath);
         void AddDynamicFile(string filePath, IEnumerable<string> folderNames = null);
-        void RemoveDynamicFile(string fullPath);
+        void RemoveDynamicFile(string filePath);
```

Comment:
Note that this is a breaking change. If this is after in be @davkean 's changes, perhaps we can live this parameter having an inconsistent name?